.gitignore

```diff
@@ -328,3 +330,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+# ignore all generated yaml files
+Minecraftpe/
```

Comment:
We probably shouldn't leave this in the .gitignore @NateRickard.  This is team specific.  Maybe just make a generic output folder and exclude that?

Reply:
Yes, was actually just adding a note re: that change

Reply:
added a /output folder and ignored that